MobiFlight/MobiFlightFirmwareUpdater.cs

```diff
@@ -107,33 +104,36 @@ public static void RunAvrDude(String Port, Board board, String FirmwareName)
 
             String FullAvrDudePath = $@"{ArduinoIdePath}\{AvrPath}";
 
-            var proc1 = new ProcessStartInfo();
-            string anyCommand = 
-                $@"-C""{FullAvrDudePath}\etc\avrdude.conf"" {verboseLevel} -p{ArduinoChip} -c{C} -P{Port} -b{Bytes} -D -Uflash:w:""{FirmwarePath}\{FirmwareName}"":i";
-            proc1.UseShellExecute = true;
-            proc1.WorkingDirectory = $@"""{FullAvrDudePath}""";
-            proc1.FileName = $@"""{FullAvrDudePath}\bin\avrdude""";
-            proc1.Arguments = anyCommand;
-            proc1.WindowStyle = ProcessWindowStyle.Hidden;
-            Log.Instance.log($"{proc1.FileName}.", LogSeverity.Debug);
-            Log.Instance.log($"{anyCommand}.", LogSeverity.Debug);
-            Process p = Process.Start(proc1);
-            if (p.WaitForExit(board.AvrDudeSettings.Timeout))
+            foreach (var baudRate in board.AvrDudeSettings.BaudRates)

```

Comment:
The meat of the change. One additional `foreach` loop to try all the baud rates. So simple!